CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- **NumericStepper** Refrains from using the label tag if there is no label, to prevent the keyboard from popping up on iOS unnecessarily
```

Comment:
I think in this case 'Refrains the use of label tag ' is better here than  'Refrains from using'.

Reply:
As far as I know, you refrain _from sth_, not refrain _sth_

Reply:
Checkout the first example https://dictionary.cambridge.org/us/dictionary/english/refrain

Reply:
Great, approved then.